examples/assets/scripts/misc/rotator.mjs

```diff
@@ -0,0 +1,15 @@
+import { Script } from 'playcanvas';
+
+class Rotator extends Script {
+    /**
+     * @attribute
```

Comment:
That will be a lot of linting fun when people start using `@assetType` instead of `@type {Asset<SomeType>}` too :sweat_smile:

Reply:
Well I think we decided on `@resource` over `@assetType` in the end

Reply:
Yea I know, we gotta whitelist those new tags now?

Reply:
We have Monaco support for this, we need to see how this works in vscode